pom.xml

```diff
@@ -22,7 +22,8 @@
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <information.model.version>4.0.0</information.model.version>
-      	<multipart.message.processor.version>1.0.6-SNAPSHOT</multipart.message.processor.version>
+      	<multipart.message.processor.version>1.0.7-SNAPSHOT</multipart.message.processor.version>
+      	<log4j.version>2.14.0</log4j.version>
```

Comment:
Same like in MultiparMessageProcessor - 2 different versions of same jar
Since this one has dependency on MMP, can you remove version from this one and try will it work?

Reply:
Removed it entirely since we now have in the MMP